newsplease/crawler/simple_crawler.py

```diff
@@ -26,14 +93,36 @@ def _fetch_url(url, is_threaded, timeout=None):
         :param timeout: in seconds, if None, the urllib default is used
         :return: html of the url
         """
-        headers = {'User-Agent': 'Mozilla/5.0'}
-        req = urllib.request.Request(url, None, headers)
-        html = urllib.request.urlopen(req, data=None, timeout=timeout).read()
-
+        html_str = None
+        # send
+        try:
+            # read by streaming chunks (stream=True, iter_content=xx)
+            # so we can stop downloading as soon as MAX_FILE_SIZE is reached
+            response = requests.get(url, timeout=timeout, verify=False, allow_redirects=True, headers=HEADERS)
```

Comment:
regarding `allow_redirect=True`, I think this is not standard behaviour of the currently used "urllib.request", or is it?

Reply:
It's default true in requests https://requests.readthedocs.io/en/master/api/
Did not find a corresponding argument https://docs.python.org/3/library/urllib.request.html